### DISCUSSION TITLE
Filtering on BB Column Populated w/ Date Picker

### DISCUSSION THREAD

**Discussion by chadgriff** [Budibase/budibase#3586] [Help] [answered]
I am having trouble determining how to structure the binding to filter records based on an "eventEnd" column which was populated via a Date Picker component.  The query filter will not save unless I have a hard coded value and I cannot get a binding setup to format a date the same way.  The desired behavior is to return all records which have an end date prior to "now".
![2021-11-29_12-29-54](https://user-images.githubusercontent.com/15015476/143915173-ec28025a-0c58-48ff-a5de-61069f1ad469.png)

**Comment by chadgriff** [comment, marked as answer]
UPDATE: I can get the following binding to work `{{date now "YYYY-MM-DDT00:00:00.000" "America/New_York" }}` when applied as a filter to a data provider within a screen. However, the same binding will not save when attempting to apply it to the "Query Rows" component of an automation.

I am running Budibase v0.9.187 on a Digital Ocean droplet using `docker-compose`

**Comment by mjashanks** [comment]
I see what you mean - I have reproduced this exactly. I have logged this as a bug: https://github.com/Budibase/budibase/issues/3598
